From 961e969af1f3bee9baadf56e52557faeb4f26550 Mon Sep 17 00:00:00 2001
From: Darren Etheridge <detheridge@ti.com>
Date: Fri, 28 Aug 2015 06:18:00 +0000
Subject: [PATCH 2/4] ARM: dts: am33xx: add DT node for gpu

Add the node into the am33xx.dtsi file for the SGX GPU.

Signed-off-by: Darren Etheridge <detheridge@ti.com>
Signed-off-by: Eric Ruei <e-ruei1@ti.com>
---
 arch/arm/boot/dts/am33xx.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index 2ab1579..687b55c 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -948,6 +948,14 @@
 			reg = <0x48310000 0x2000>;
 			interrupts = <111>;
 		};
+
+		sgx: sgx@0x56000000 {
+			compatible = "ti,am335x-sgx530", "img,sgx530";
+			ti,hwmods = "gfx";
+			reg = <0x56000000 0x1000000>;
+			interrupts = <37>;
+			status = "okay";
+		};
 	};
 };
 
-- 
2.5.1

